{"name": "Проверка ошибки 401 при получении заказов без авторизации", "status": "passed", "steps": [{"name": "Получение заказов пользователя", "status": "passed", "parameters": [{"name": "access_token", "value": "''"}], "start": 1743958730406, "stop": 1743958730551}], "start": 1743958730406, "stop": 1743958730552, "uuid": "39555dbb-c446-4c6c-acfd-64b07a3b2c20", "historyId": "f384f6e44b534b6570905edd0ab1ebdf", "testCaseId": "f384f6e44b534b6570905edd0ab1ebdf", "fullName": "tests.test_get_users_orders.TestGetUserOrders#test_get_orders_without_auth_returns_401", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_users_orders"}, {"name": "subSuite", "value": "TestGetUserOrders"}, {"name": "host", "value": "elishakh"}, {"name": "thread", "value": "18472-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_users_orders"}]}